packages/web-components/src/components/cbp-app-header/cbp-app-header.tsx

```diff
@@ -7,9 +7,41 @@ import { Component, Host, h, Prop } from '@stencil/core';
 export class CbpAppHeader {
 
   
-  /** Specifies the context of the component as it applies to the visual design and whether it inverts when light/dark mode is toggled. Default behavior is "light-inverts" and does not have to be specified. */
-  @Prop({ reflect: true }) context: "light-inverts" | "light-always" | "dark-inverts" | "dark-always";
+  private navItems: HTMLCbpNavItemElement[] = [];
 
+  @Element() host: HTMLElement;
+
+  initNavItemset() {
+    // check for a default navItem, otherwise set the first one active
+    let activeNavItem;
+    this.navItems.forEach(navItem => {
+      if (navItem.selected === true) activeNavItem = navItem;
+    });
+    this.setActiveNav(activeNavItem);
+  }
+
+  setActiveNav(activatedNav) {
```

Comment:
this should probably be called on componentDidLoad as well. I'm not seeing the aria-current on the home link.

Reply:
updated to call this in componentDidLoad